src/services/users.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const { generateId } = require('../functions/generateId');
+const { User } = require('../models/users');
+
+const normalize = ({ id, name }) => {
+  return {
+    id, name,
+  };
+};
```

Comment:
Intead of using this normalize, you can specify needed attributes in [select query](https://sequelize.org/docs/v6/core-concepts/model-querying-basics/#specifying-attributes-for-select-queries).